homeworks/es6-continued/yuracr9/promisify.js

```diff
@@ -0,0 +1,43 @@
+const myUrl = 'https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2017/master/homeworks/es6-continued/response.json';
+
+// function for requesting data from the server
+function getUrl(url, cb) {
+  const xhr = new XMLHttpRequest();
+
+  xhr.open('GET', url, true);
+  xhr.send();
+
+  xhr.onreadystatechange = function () {
+    if (this.readyState != 4) return;
+
+    if (this.status != 200) {
+      return cb(new Error(this.statusText));
+    }
+
+    return cb(null, this.responseText);
+  }
+}
+
+function promisify(func) {
```

Comment:
This promisify fn will accept only one argument. Try to make possible put many arguments in promisified fn and make another PR

Reply:
> put many arguments

@komarnitskyi there's no need for that. [Check out the task](https://github.com/kottans/chernivtsi-frontend-2017/tree/master/homeworks/es6-continued#task-3-promisify).

> make another PR

Just push new commits in this PR.